software/bracket.yml

```diff
@@ -0,0 +1,13 @@
+name: "Bracket"
+website_url: "https://evroon.github.io/bracket"
+source_code_url: "https://github.com/evroon/bracket"
+description: "Flexible tournament system to build a tournament setup, add teams, schedule matches, keep track of scores and present ranking live to the public."
+licenses:
+  - MIT
+platforms:
+  - Python
+  - Javascript
+  - Docker
```

Comment:
> https://github.com/evroon/bracket#quickstart

```suggestion
platforms:
  - Docker
  - Nodejs
```

Reply:
Ah apparently I misunderstood [this section](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#add-a-languageplatform), I thought the programming language the tool runs on is also part of this list, but it's only about what is needed to build/run the software I see now.